DllImportGenerator/TestAssets/NativeExports/Arrays.cs

```diff
@@ -120,5 +121,18 @@ public static void Append(int** values, int numOriginalValues, int newValue)
             newArray[numOriginalValues] = newValue;
             *values = newArray;
         }
+
+        [UnmanagedCallersOnly(EntryPoint = "and_all_members")]
+        [DNNE.C99DeclCode("struct bool_struct;")]
```

Comment:
Up to you, but support exists for defining a header file and then supplying include paths - see [`DnneAdditionalIncludeDirectories`](https://github.com/AaronRobinsonMSFT/DNNE/blob/master/src/msbuild/DNNE.props).